### ISSUE TITLE
[ERROR]

### ISSUE THREAD

**Issue by shadowglost** [victorchall/EveryDream2trainer#225]
Hi, I left the IA training overnight, there was a blackout in my neighborhood interrupting the training, and when I tried to restart the training in the morning, this error appeared:
Microsoft Windows [versão 10.0.19044.3086]
(c) Microsoft Corporation. Todos os direitos reservados.

C:\Area de IA\EveryDream2trainer-main>activate_venv.bat

C:\Area de IA\EveryDream2trainer-main>call venv/scripts/activate.bat

(venv) C:\Area de IA\EveryDream2trainer-main>python train.py -- config train.json
A matching Triton is not available, some optimizations will not be enabled.
Error caught was: No module named 'triton'
Loading training config from None.
Error on loading training config from None.
usage: train.py [-h] [--amp] [--batch_size BATCH_SIZE] [--ckpt_every_n_minutes CKPT_EVERY_N_MINUTES]
                [--clip_grad_norm CLIP_GRAD_NORM] [--clip_skip {0,1,2,3,4}] [--cond_dropout COND_DROPOUT]
                [--data_root DATA_ROOT] [--disable_amp] [--disable_textenc_training] [--disable_unet_training]
                [--disable_xformers] [--flip_p FLIP_P] [--gpuid GPUID] [--gradient_checkpointing]
                [--grad_accum GRAD_ACCUM] [--logdir LOGDIR] [--log_step LOG_STEP] [--lowvram] [--lr LR]
                [--lr_decay_steps LR_DECAY_STEPS] [--lr_scheduler {constant,linear,cosine,polynomial}]
                [--lr_warmup_steps LR_WARMUP_STEPS] [--max_epochs MAX_EPOCHS] [--no_prepend_last] [--no_save_ckpt]
                [--optimizer_config OPTIMIZER_CONFIG] [--plugins PLUGINS [PLUGINS ...]] [--project_name PROJECT_NAME]
                [--resolution {256,320,384,448,512,576,640,704,768,832,896,960,1024,1088,1152,1280,1536}]
                --resume_ckpt RESUME_CKPT [--run_name RUN_NAME] [--sample_prompts SAMPLE_PROMPTS]
                [--sample_steps SAMPLE_STEPS] [--save_ckpt_dir SAVE_CKPT_DIR]
                [--save_every_n_epochs SAVE_EVERY_N_EPOCHS] [--save_ckpts_from_n_epochs SAVE_CKPTS_FROM_N_EPOCHS]
                [--save_full_precision] [--save_optimizer] [--seed SEED] [--shuffle_tags] [--useadam8bit] [--wandb]
                [--validation_config VALIDATION_CONFIG] [--write_schedule] [--rated_dataset]
                [--rated_dataset_target_dropout_percent RATED_DATASET_TARGET_DROPOUT_PERCENT]
                [--zero_frequency_noise_ratio ZERO_FREQUENCY_NOISE_RATIO] [--enable_zero_terminal_snr]
                [--load_settings_every_epoch] [--min_snr_gamma MIN_SNR_GAMMA] [--ema_decay_rate EMA_DECAY_RATE]
                [--ema_strength_target EMA_STRENGTH_TARGET] [--ema_update_interval EMA_UPDATE_INTERVAL]
                [--ema_device EMA_DEVICE] [--ema_sample_nonema_model] [--ema_sample_ema_model]
                [--ema_resume_model EMA_RESUME_MODEL]
train.py: error: the following arguments are required: --resume_ckpt

On both attemps I was using the  "resume_ckpt panopstor/EveryDream" as instructed in the "[Download and setup base models](https://github.com/victorchall/EveryDream2trainer/blob/main/doc/BASEMODELS.md)".

Seams to me that it created some file during the interrupted attemp and now wants to resume that, problem is, if there really is a file I did not found it on my pc (running local) any tips on how to locate this file or how to start from scratch? Deleting the EveryDream2trainer-main folder did not fixed.

**Comment by victorchall** [comment]
```python train.py -- config train.json```

There shouldn't be a space after the two dashes.
